tests/UKHO.ERPFacade.API.FunctionalTests/Model/JsonInputPriceChangeHelper.cs

```diff
@@ -1,12 +1,16 @@
-﻿using Newtonsoft.Json;
-
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
 
 namespace UKHO.ERPFacade.API.FunctionalTests.Model
 {
-    public class UoSInputJSONHelper
+    public class JsonInputPriceChangeHelper
     {
         [JsonProperty("corrid")]
-        public string Corrid { get; set; }
+        public string? Corrid { get; set; }
```

Comment:
Does this need to be nullable ?

Reply:
Yes this field should be null for PriceChange, as it was mentioned a rule. And corrId will get generated at Runtime dynamically

Reply:
Yes